tools/autograd/derivatives.yaml

```diff
@@ -731,6 +731,12 @@
   other: grad * other / result
   result: self_t * self_p / result + other_t * other_p / result
 
+- name: special_fresnel_integral_c(Tensor self) -> Tensor
+  self: non_differentiable
```

Comment:
`gradient * cos(PI * (self * self) / 2)`

Reply:
@mruberry A good example of where constants (i.e., `PI`) would be useful.

Reply:
@albanD do we not support constants in derivatives.yaml?

Reply:
@albanD Good to know!